pkg/noun/allocate.h

```diff
@@ -144,8 +144,9 @@ STATIC_ASSERT( u3a_vits <= u3a_min_log,
 
         c3_w off_w;                           //  spin stack offset
         c3_w fow_w;                           //  spin stack overflow count
+        u3p(u3h_root) lop_p;                  //  %loop hint set
```

Comment:
This has to be initialized on home road for existing piers. A real migration, or just a zero-check in find-home.